gnucash/gnome-utils/account-quickfill.c

```diff
@@ -176,6 +187,8 @@ build_shared_quickfill (QofBook *book, Account *root, const char * key,
     qfb->load_list_store = TRUE;
     qfb->list_store = gtk_list_store_new (NUM_ACCOUNT_COLUMNS,
                                           G_TYPE_STRING, G_TYPE_POINTER);
+    qfb->list_store_full = gtk_list_store_new (NUM_ACCOUNT_COLUMNS,
+                                          G_TYPE_STRING, G_TYPE_POINTER);
```

Comment:
Please fix the indentation so that the parameters line up throughout your changes.